src/aspire/source/__init__.py

```diff
@@ -106,6 +106,8 @@ def __init__(self, L, n, dtype='double', metadata=None, memory=None):
                     degrees=True
                 )
 
+        self.unique_filters = None
+        self.filter_indices = None
```

Comment:
Why do you have an attribute `filter_indices` here when you have a property by the identical name?

Reply:
Removed in new version. It was the old way to initialize as None.